src/assemble_workflow/dist.py

```diff
@@ -29,6 +32,10 @@ def __extract__(self, dest: str) -> None:
     def __build__(self, name: str, dest: str) -> None:
         pass
 
+    #@property
```

Comment:
remove

Reply:
Removed, this is due to a temp test before moving the self.distribution as an argument in dists.py.